python_examples/rebound.py

```diff
@@ -1,4 +1,9 @@
 from ctypes import *
+import numpy
```

Comment:
Hm. I don't like that it uses numpy. I think `math` should be able to do the same.